src/w3iProxy/notifyProviders/internalNotifyProvider.ts

```diff
@@ -106,8 +103,8 @@ export default class InternalNotifyProvider implements W3iNotifyProvider {
 
   // ------------------- Method-forwarding for NotifyClient -------------------
 
-  public isInitialSubscriptionLoadComplete() {
-    return this.gotFirstWatchSubscriptionsResponse;
+  public hasFinishedInitialLoad() {
+    return this.notifyClient?.hasFinishedInitialLoad() || false;
```

Comment:
Do we need `|| false` here? If yes, do we have a case that `this.notifyClient?.hasFinishedInitialLoad()` always falsy?

Reply:
just added the `|| false` instead `notifyClient` is `undefined`

Reply:
@devceline I mean that on the Notify side, if somehow it's not set to true, we'll still show skeletons infinitely, right?

Reply:
Yes, but that should never ever happen. We set this to true even if there an error

Reply:
Understood. We should always handle all cases. Wanted to double check